frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -416,10 +428,11 @@ function ReaderDictionary:onLookupWord(word, is_sane, boxes, highlight, link, tw
     logger.dbg("dict stripped word:", word)
 
     self.highlight = highlight
+    local disable_fuzzy_search = self.ui.doc_settings and self.disable_fuzzy_search or self.disable_fuzzy_search_fm
```

Comment:
Same here?
That is, if we toggled it to true then false (fuzzy search enabled), we would use the global setting?